<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowFormulaEditor</class>
 <widget class="QDialog" name="WindowFormulaEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="group_WriteToFormula">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>561</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Add to formula:</string>
   </property>
   <widget class="QPushButton" name="btn_AddPlus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string notr="true">+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_AddMinus">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string notr="true">-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_AddMultiply">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string notr="true">*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_AddDivide">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string notr="true">/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_AddOpenBracket">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string notr="true">(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_AddCloseBracket">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string notr="true">)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_AddValueToList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add value to list</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Values">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Values:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_AddOperand">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>110</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_AddAbs">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string notr="true">Abs</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lb_Variables">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>331</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="group_CurrentFormula">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>561</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Current formula:</string>
   </property>
   <widget class="QLabel" name="lbl_Name">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_FormulaName">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_CurrentFormula">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Current formula</string>
    </property>
    <property name="text">
     <string notr="true">(A+B)/C</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Equation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Equation:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_SaveFormula">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ClearFormula">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_LoadFormula">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Evaluate">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Evaluate</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
